server/service/src/stocktake_line/update/mod.rs

```diff
@@ -4,26 +4,10 @@ mod validate;
 pub use validate::validate;
```

Comment:
Not actually related to your changes, but I get a warning because this validate and the delete validate are both pub... can we make private? 🙏